### PR TITLE
Test Taker API parameter rename and validation

### PR DESCRIPTION
**Pull request by augustas**
https://oat-sa.atlassian.net/browse/TAO-7789

### CODE REVIEW

File: doc/swagger.json [hunk 1]
```diff
@@ -97,11 +97,11 @@
             "description": "Password",
             "type": "string"
           },
-          "uiLg": {
+          "userLanguage": {
```
Comment (peetya) [inline doc/swagger.json]: It should be marked as required. What do you think?

File: actions/RestTestTakers.php [hunk 2]
```diff
@@ -110,12 +110,12 @@ class RestTestTakers extends \tao_actions_CommonRestModule
      *         description="Password"
      *     ),
      *     @OA\Property(
-     *         property="uiLg",
+     *         property="userLanguage",
```
Comment (peetya) [inline actions/RestTestTakers.php]: It should be marked as required. What do you think?

File: doc/swagger.json (hunk 1 above)
Comment (augustas) [reply doc/swagger.json]: You're right, I'll update the documentation :+1:

File: actions/RestTestTakers.php (hunk 2 above)
Comment (augustas) [reply actions/RestTestTakers.php]: You're right, I'll update the documentation :+1:

**Pull request merged**